aiida/cmdline/commands/cmd_calcjob.py

```diff
@@ -176,8 +176,8 @@ def calcjob_inputls(calcjob, path, color):
 
     try:
         list_repository_contents(calcjob, path, color)
-    except ValueError as exception:
-        echo.echo_critical(exception)
+    except FileNotFoundError:
```

Comment:
the only comment I have is that this might become a different exception when we move to the new repository implementation.
is there perhaps a higher-level exception that we could catch here?

Reply:
The tests I have added will then exactly catch that change and we can adapt it. Don't see the point in trying to anticipate the type of exception that this might throw in the future. Or did you mean to catch a more generic exception like `OSError` which is the [base class](https://docs.python.org/3/library/exceptions.html#FileNotFoundError) of `FileNotFoundError`?

Reply:
right, I meant more generic. 
Anyhow, if you think we should just use the one we have currently it's also ok. the tests will fail if this changes, so it will be fixed then.